packages/frontend/component/src/ui/tooltip/tooltip.stories.tsx

```diff
@@ -18,6 +20,75 @@ const Template: StoryFn<TooltipProps> = args => (
 export const Default: StoryFn<TooltipProps> = Template.bind(undefined);
 Default.args = {};
 
+export const WithShortCut = () => {
+  const shortCuts = [
+    ['Text', 'T'],
+    ['Bold', ['⌘', 'B']],
+    ['Quick Search', ['⌘', 'K']],
```

Comment:
Can you please make the shortcust a big smarter, like `$mod` or `CommandOrControl` -> `⌘`/`CTRL`?

Reply:
But is it possible to access `environment` in `@affine/component` ?

Reply:
maybe not, but I guess we can have an inplace check for Mac instead of relying on `enviorment`